### PR TITLE
Announce membership applications on the Changelog

### PR DESCRIPTION
**Pull request by galgalesh**
Given not all members always read the meeting minutes, it's best to explicitly ask prospective members to announce their application on the Changelog. This way, everyone who keeps somewhat up-to-date with space matters will see who wants to become a member.

This PR also attempts to further clarify the existing steps.

### CODE REVIEW

**Review by evils — COMMENTED** [review]
ensuring everyone knows there's a new applicant is important, thanks for clarifying that

File: 5-membership.md [hunk 1]
```diff
@@ -3,16 +3,18 @@
...
-2. Attend a meeting and present your application. On this meeting you announce you want to become a member and who your sponsors are. If you want you can talk a bit about why you're interested in the space. *Note: this meeting is simply an announcement. Your membership won't be discussed further, and no decision about it will be made on that meeting.*
-3. On the next meeting, your membership will be decided upon. It's useful to have you present at that meeting, but this is not a requirement. As a bonus for coming to the meeting, membership is often the first thing to be decided on a meeting. When you get voted in, you can join the rest of the meeting as a member! Membership decisions at a meeting follow the same process as other meeting decisions. See Chapter 3 for more info.
-4. Create an automatic bank transfer to transfer the membership fee each month. You will get your key after you made your first payment.
+2. Announce your membership application on the Changelog. Include in the message who your sponsors are.
```
[the remark this answers is not in the document]
Comment (galgalesh) [reply 5-membership.md:6]: They should do it themselves, in my opinion, but it doesn't matter much. The important part is that there is an announcement. I wouldn't change the text here to keep it simple.

[the remark this answers is not in the document]
Comment (evils) [reply 5-membership.md:6]: someone else doing it on their behalf could be seen as them doing it
(i was mostly wondering about implicitly setting a requirement that they're reachable via our chat platform)

[the remark this answers is not in the document]
Comment (galgalesh) [reply 5-membership.md:6]: Hm, it might be a good implicit requirement, indeed.

**Review by evils — APPROVED** [review]

**Pull request merged**